File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    n = get_int("height: ")
    if n > 0 and n < 9:
        break

# loop to handle number of rows
for i in range(n):
    # k no of spaces
    k = 1
    # loop to handle spaces
    for k in range(n - i - 1):
        print(" ", end="")
    # loop to handle number of columns
    for j in range(i+1):
        # printing #
        print("#", end="")
    # printing newline after each row
    print()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

// function prototype
void mario_more();

// main
int main()
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while
    (n < 1 || n > 8);

    mario_more(n);

    return 0;
}

void mario_more(int n)
{
    for (int i = 0; i < n; i++)
    {
        // Loop to add spaces to the left mario
        for (int k = n - i; k > 1; k--)
        {
            printf(" ");
        }
        // loop to add  the left mario #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        // loop to add 2 spaces after the #
        for (int z = 0; z < 2; z++)
        {
            printf(" ");
        }
        // loop to print the right mario
        for (int y = 0; y <= i; y++)
        {
            printf("#");
        }
        printf("\n");
    }
}